From: Laurent Declercq <l.declercq@nuxwin.com>
Date: Mon, 29 Oct 2018 03:16:19 +0100
Subject: shared-lib-ext

---
 Configure | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Configure b/Configure
index 22953fb..dbaed5b 100755
--- a/Configure
+++ b/Configure
@@ -1855,7 +1855,8 @@ while (<IN>)
 	elsif ($shared_extension ne "" && $shared_extension =~ /^\.s([ol])\.[^\.]*\.[^\.]*$/)
 		{
 		my $sotmp = $1;
-		s/^SHARED_LIBS_LINK_EXTS=.*/SHARED_LIBS_LINK_EXTS=.s$sotmp.\$(SHLIB_MAJOR) .s$sotmp/;
+#		s/^SHARED_LIBS_LINK_EXTS=.*/SHARED_LIBS_LINK_EXTS=.s$sotmp.\$(SHLIB_MAJOR) .s$sotmp/;
+		s/^SHARED_LIBS_LINK_EXTS=.*/SHARED_LIBS_LINK_EXTS=.s$sotmp/;
 		}
 	elsif ($shared_extension ne "" && $shared_extension =~ /^\.[^\.]*\.[^\.]*\.dylib$/)
 		{
